content/v3/repos/releases.md

```diff
@@ -57,7 +57,13 @@ View the latest published release for the repository.
 
 ## Get a release by tag name
 
-Get a release with the specified tag. Users must have push access to the repository to view draft releases.
+Get a release for the specified git tag.
+
+{{#tip}}
+
+Releases that are drafts usually don't have their git tag created yet, so they will have to be looked up by listing all releases for the repository. Only users with push access will receive listings for draft releases.
```

Comment:
A draft has a "desired" tag name, i.e. the tag name that is to be automatically created in git when the release is published. This desired tag name will be the value of "tag_name" property in API payload. The random tag name `untagged-9ffe338327c4c21247a3` is not a value for any property in release API payload, and occurs only in `html_url` for the release.

However, I just checked and it seems that the draft release *is* accessible by providing its random tag name to the tags lookup instead of the "desired" tag name, so thanks for that hint. But now that I think about it, this is surprising behavior and I would classify this as a bug. I would expect that when doing tag-based lookup, a release should get found based on its `tag_name`, even when it's a draft release and regardless of the fact whether the tag exists pyhsically in git or not.